packages/drawer/__tests__/Drawer.test.tsx

```diff
@@ -52,18 +52,70 @@ describe('with props.children', () => {
   })
 })
 
-describe('with props.ariaLabelledBy', () => {
+describe('with props.ariaLabel', () => {
   it('renders labelled drawer', () => {
     const label = 'Hello'
-    render(
-      <Drawer ariaLabelledBy="drawer-title" open={true}>
-        <h1 id="drawer-title">{label}</h1>
-      </Drawer>,
-    )
+    render(<Drawer ariaLabel={label} open={true} />)
```

Comment:
Possible to make this prop `aria-label`? I think that is the default prop that React uses

Reply:
Prop renamed in 851232319411d5a1a4bded521cd4d314184c7f76